docs/website/src/main/resources/certifications/jakarta-websocket/1.1/TCK-Results.md

```diff
@@ -0,0 +1,35 @@
+TCK Results
+===========
+
+As required by the
+[Eclipse Foundation Technology Compatibility Kit License](https://www.eclipse.org/legal/tck.php),
+following is a summary of the TCK results for releases of Jakarta WebSocket.
+
+# Tyrus 1.15, as delivered in Eclipse GlassFish 5.1, Full Profile, Certification Request
+
+- Product Name, Version and download URL (if applicable) \
+  Eclipse GlassFish 5.1, includes: WebSocket 1.1.1 with Tyrus 1.15
+  [Eclipse GlassFish 5.1](https://www.eclipse.org/downloads/download.php?file=/glassfish/glassfish-5.1.0.zip)
```

Comment:
These results should be hosted by the Tyrus project.

Reply:
done